Header header

float64 latitude
float64 longitude
int8 satellites
float64 utc